entities/entities/gunlab/shared.lua

```diff
@@ -1,11 +1,9 @@
-ENT.Type = "anim"
-ENT.Base = "base_gmodentity"
+ENT.Base = "lab_base"
 ENT.PrintName = "Gun Lab"
-ENT.Author = "Pcwizdan"
-ENT.Spawnable = false
-ENT.CanSetPrice = true
 
-function ENT:SetupDataTables()
-    self:NetworkVar("Int", 0, "price")
-    self:NetworkVar("Entity", 1, "owning_ent")
+function ENT:initVars()
```

Comment:
I did it this way because I think DarkRPMod loads language files after entities are loaded which would cause issues with the DarkRP.getPhrase call. Same with the config.